apps/dsp-app/src/app/workspace/resource/properties/properties.component.ts

```diff
@@ -567,6 +553,34 @@ export class PropertiesComponent implements OnInit, OnChanges, OnDestroy {
     localStorage.setItem('showAllProps', JSON.stringify(this.showAllProps));
   }
 
+  private _getResourceAttachedData() {
+    this._store.dispatch([
+      new GetAttachedUserAction(this.resource.res.id, this.resource.res.attachedToUser),
+      new GetAttachedProjectAction(this.resource.res.id, this.resource.res.attachedToProject),
+    ]);
+
+    this.project = this._store
+      .selectSnapshot(ProjectsSelectors.allProjects)
+      .find(p => p.id === this.resource.res.attachedToProject);
+    this._actions$
```

Comment:
This seems obsolete and wrong. Why is this project set in two different ways once to a ReadProject and then to a StoredProject?

Reply:
project is provided from multiple endpoints. One of them has StoredProject, other has ReadProject which extends StoredProject. It would be nice, if API can provide single type.